src/zenml/cli/base.py

```diff
@@ -219,54 +219,59 @@ def go() -> None:
         gave_email = _prompt_email()
         metadata = {"gave_email": gave_email}
 
-    # Add telemetry
-    track_event(AnalyticsEvent.RUN_ZENML_GO, metadata=metadata)
-
-    console.print(zenml_go_privacy_message, width=80)
-
-    zenml_tutorial_path = os.path.join(os.getcwd(), "zenml_tutorial")
-
-    if not os.path.isdir(zenml_tutorial_path):
-        try:
-            from git.repo.base import Repo
-        except ImportError as e:
-            logger.error(
-                "At this point we would want to clone our tutorial repo onto "
-                "your machine to let you dive right into our code. However, "
-                "this machine has no installation of Git. Feel free to install "
-                "git and rerun this command. Alternatively you can also "
-                f"download the repo manually here: {TUTORIAL_REPO}. The "
-                f"tutorial is in the 'examples/quickstart/notebooks' directory."
-            )
-            raise GitNotFoundError(e)
-
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            tmp_cloned_dir = os.path.join(tmpdirname, "zenml_repo")
-            with console.status(
-                "Cloning tutorial. This sometimes takes a minute..."
-            ):
-                Repo.clone_from(
-                    TUTORIAL_REPO,
-                    tmp_cloned_dir,
-                    branch=f"release/{zenml_version}",
+    with event_handler(AnalyticsEvent.RUN_ZENML_GO) as handler:
```

Comment:
I would suggest naming this `analytics_handler` or something similar, `handler` lacks any meaning in the code further down, especially for these long indents

Reply:
In the cases, where the metadata is already ready, I have put the metadata in the constructor and removed the `handler`. In all the other cases where the metadata is computed halfway through the operation, I have renamed the `handler` to `analytics_handler` following your suggestion.